### PR TITLE
docs(aiplatform): Add PredictTextClassificationSample #2396

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

**Comment by meteatamel** [comment]
@hemanshv I think you missed to review this one?

**Review by hemanshv — CHANGES_REQUESTED** [review]

File: aiplatform/api/AIPlatform.Samples.Tests/PredictTextClassificationTest.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    private readonly PredictTextClassificationSample _sample;
+
+    public PredictTextClassificationTest(AIPlatformFixture fixture)
+    {
+        _fixture = fixture;
+        _sample = new PredictTextClassificationSample();
+    }
+
+    [Fact]
+    public void Runs()
```
[the remark this answers is not in the document]
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples.Tests/PredictTextClassificationTest.cs:32]: Ack

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START aiplatform_sdk_classify_news_items]
+
+using Google.Cloud.AIPlatform.V1;
+using wkt = Google.Protobuf.WellKnownTypes;
```
[the remark this answers is not in the document]
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:20]: There's another `Value` in `Google.Cloud.AIPlatform.V1`, that's why we need to refer to the right one but `using Value = Google.Protobuf.WellKnownTypes.Value;` would be a better way of referring.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        {
+            Endpoint = $"{locationId}-aiplatform.googleapis.com"
+        }.Build();
+
+        // Configure the parent resource.
+        var endpoint = EndpointName.FromProjectLocationPublisherModel(projectId, locationId, publisher, model);
+
+        // Initialize request argument(s).
+        var content = @"What is the topic for a given news headline?
+- business
```
[the remark this answers is not in the document]
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:46]: You mean align `-business` and the rest of the text with line 45? That'd introduce extra spaces in the string though, right? This is a multi-line string.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+Text: CNBC Reports Rising Digital Profit as Print Advertising Falls
+The answer is:";
+
+        var instances = new List<wkt::Value>
+        {
+            wkt::Value.ForStruct(new()
+            {
+                Fields =
+                {
+                    ["content"] = wkt::Value.ForString(content),
```
[the remark this answers is not in the document]
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:76]: It's an open question whether we should link to docs from these samples. These samples will be embedded in the docs, so I don't think it's needed but we'll let Amanda decide.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+                { "topP", new wkt::Value { NumberValue = 0 } },
+                { "topK", new wkt::Value { NumberValue = 1 } }
+            }
+        });
+
+        // Make the request.
+        var response = client.Predict(endpoint, instances, parameters);
+
+        // Parse and return the content.
+        var responseContent = response.Predictions[0].StructValue.Fields["content"].StringValue;
```
[the remark this answers is not in the document]
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:96]: Ack

File: aiplatform/api/AIPlatform.Samples.Tests/PredictTextClassificationTest.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (amanda-tarafa) [reply aiplatform/api/AIPlatform.Samples.Tests/PredictTextClassificationTest.cs:32]: Yes please, thatnks! Here, and elsewhere.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs (hunk 3 above)
[the remark this answers is not in the document]
Comment (amanda-tarafa) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:46]: Yes, we don't want to align these, as it's a multiline string, those spaces would show up on the string value itself.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs (hunk 2 above)
[the remark this answers is not in the document]
Comment (amanda-tarafa) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:20]: As in other PRs, using an alias would be best.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs (hunk 4 above)
[the remark this answers is not in the document]
Comment (amanda-tarafa) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:76]: If these are all explained in the docs where the samples are, then that's enough.

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs (hunk 5 above)
[the remark this answers is not in the document]
Comment (amanda-tarafa) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs:96]: Yes, maybe. Specially @meteatamel if you are following my suggestion elsewhere to show sorted predictions based on score, then using linq might be cleaner.

File: aiplatform/api/AIPlatform.Samples.Tests/PredictTextClassificationTest.cs [hunk 6]
```diff
@@ -0,0 +1,37 @@
+/*
```
Comment (hemanshv) [inline aiplatform/api/AIPlatform.Samples.Tests/PredictTextClassificationTest.cs:1]: placeholder to undo request changes

File: aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs (hunk 5 above)
[the remark this answers is not in the document]
Comment (meteatamel) [reply aiplatform/api/AIPlatform.Samples/PredictTextClassificationSample.cs]: I use Linq to get first but there's no score or anything like that really

**Comment by meteatamel** [comment]
@hemanshv Let me know if anything else is needed here.

**Pull request merged**